node/resolvers/Mutations/Users.ts

```diff
@@ -368,6 +368,66 @@ const Users = {
     )
   },
 
+  removeUserWithEmail: async (
+    _: void,
+    { orgId, costId, email }: UserArgs,
+    ctx: Context
+  ) => {
+    const {
+      clients: { events, storefrontPermissions: storefrontPermissionsClient },
+      vtex: { logger },
+    } = ctx as any
+
+    const variables = {
+      organizationId: orgId,
+      costCenterId: costId,
+      search: email,
+    }
+
+    storefrontPermissionsClient
+      .listUsersPaginated(variables)
+      .then((result: any) => {
+        const user = result.data.listUsersPaginated.data.find(
+          (u: any) => u.email === email
```

Comment:
what happens if the `listUsersPaginated` returns a valid response but there is no match? The API will return 200? Should we return an error maybe, indicating the user does not exists for that org/cc?

Reply:
I left it returning 200, because if the user does not exist for that cost center or organization, then technically it is ok, with the request related.